synapse/tests/utils.py

```diff
@@ -86,6 +86,14 @@
 # Default LMDB map size for tests
 TEST_MAP_SIZE = s_const.gibibyte
 
+print('Checking debug mode')
+_SYN_ASYNCIO_DEBUG = False
+# Check if DEBUG mode was set https://docs.python.org/3/library/asyncio-dev.html#debug-mode
+if (s_common.envbool('PYTHONASYNCIODEBUG') or s_common.envbool('PYTHONDEVMODE') or sys.flags.dev_mode):  # pragma: no cover
+    _SYN_ASYNCIO_DEBUG = True
+    print('found debug mode')
```

Comment:
Do we still need the extra "checking/found" messages here? seems like just displaying the value of _SYN_ASYNCIO_DEBUG is sufficient.

Reply:
I'll revert that associated change.